commands/fetch_templates_test.go

```diff
@@ -35,21 +36,28 @@ func Test_PullTemplates(t *testing.T) {
 
 // setupLocalTemplateRepo will create a local copy of the core OpenFaaS templates, this
 // can be refered to as a local git repository.
-func setupLocalTemplateRepo(t *testing.T) string {
+func setupLocalTemplateRepo(t *testing.T, useMock bool) string {
```

Comment:
What code calls this with "false" ?

Reply:
The current repo tests against `github.com/openfaas/templates`, that's why I let the possibility to get this current behaviour. I think we do not this, tests should always use mock.

Should I remove the parameter?

Reply:
Could we avoid the zip and generate the git folder instead?
```
git init
echo etc > ./template.yml
git commit -m "Initial"
```

etc?

Or add the .git folder as a sub-module?